lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -382,24 +393,34 @@ export default function getRecipeInterface(
             logDebugMessage("refreshSession: Started");
```

Comment:
you need to check for older session and clear the cookies from there if relevant